lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -179,6 +179,7 @@ def add_card(post, credit_card)
         }
 
         card.delete_if{|k,v| v.blank? }
+        card[:holderName] ||= 'Not Provided' if credit_card.is_a?(NetworkTokenizationCreditCard)
```

Comment:
This won't overwrite if the name is an empty string, not sure if that's possible, but worth looking into.

Reply:
It will because of the line above, which removes all `blank?` values (and their keys) from the `card` hash.

Reply:
👍